client/layout/masterbar/woo-core-profiler.tsx

```diff
@@ -62,17 +65,19 @@ const WooCoreProfilerMasterbar = ( { translate }: { translate: ( text: string )
 							</a>
 						</li>
 						<li className="masterbar__woo-nav-item">
-							{ shouldShowNoThanks && typeof redirectTo === 'string' && redirectTo.length && (
-								<Button
-									onClick={ () => {
-										recordTracksEvent( 'calypso_jpc_wc_coreprofiler_skip' );
-										window.location.href = redirectTo;
-									} }
-									className="masterbar__no-thanks-button"
-								>
-									{ translate( 'No, Thanks' ) }
-								</Button>
-							) }
+							{ shouldShowNoThanks &&
+								typeof redirectTo === 'string' &&
+								validUrl.isWebUri( redirectTo ) && (
+									<Button
+										onClick={ () => {
+											recordTracksEvent( 'calypso_jpc_wc_coreprofiler_skip' );
+											window.location.href = redirectTo;
```

Comment:
Sounds like a perfect candidate for `href=`, this would make the button more accessible. This button supports `href`.